roles/wazuh/ansible-wazuh-agent/templates/var-ossec-etc-ossec-agent.conf.j2

```diff
@@ -290,12 +258,10 @@
       <max_eps>{{ wazuh_agent_config.syscheck.sync_max_eps }}</max_eps>
     </synchronization>
   </syscheck>
-  {% endif %}
-
 
-  {% if ansible_system == "Linux" and wazuh_agent_config.vuls.disable == 'no' %}
```

Comment:
As mentioned in https://github.com/wazuh/wazuh-ansible/issues/464 Vuls is deprecated, this section will be removed in the future.

https://documentation.wazuh.com/3.4/user-manual/capabilities/vuls.html

Reply:
Thanks, pushed another commit.